ugvc/pipelines/train_models_pipeline.py

```diff
@@ -31,7 +31,7 @@
 from ugvc import logger
 from ugvc.filtering import transformers, variant_filtering_utils
 from ugvc.filtering.tprep_constants import GtType, VcfType
-from ugvc.utils import misc_utils
+from ugbio_core.metrics_utils import read_hdf
```

Comment:
read_hdf was moved to metrics_utils in ugbio_core:
https://github.com/Ultimagen/ugbio-utils/blob/7cd16e1ca18c33e3a1c65d6f841dbbbd31b9bd63/src/core/ugbio_core/metrics_utils.py#L90